_config.yml

```diff
@@ -1,5 +1,5 @@
 # Plugins
-gems:
```

Comment:
Legit change or typo?

Reply:
Yeah, `gems` is deprecated in favour of `plugins`